app/api/feedback/route.ts

```diff
@@ -0,0 +1,30 @@
+import { addFeedback } from "@/lib/firebase/firestore";
+import { NextRequest, NextResponse } from "next/server";
+
+export async function POST(req: NextRequest) {
+  try {
+    const data = await req.json();
+
+    if ((data.rating < 0 || data.rating > 5) && data.comment.trim() === "") {
+      throw new Error("Invalid input");
+    }
```

Comment:
This will throw 500 error to frontend, not a good practice kub. Try throwing 400 invalid request instead. 
```ts
return NextResponse.json(..., { status: 400 })
```